hikari/impl/entity_factory.py

```diff
@@ -265,6 +266,12 @@ def __init__(self, app: traits.RESTAware) -> None:
             base_interactions.InteractionType.APPLICATION_COMMAND: self.deserialize_command_interaction,
             base_interactions.InteractionType.MESSAGE_COMPONENT: self.deserialize_component_interaction,
         }
+        self._interaction_option_type_mapping: typing.Dict[int, typing.Callable[[typing.Any], typing.Any]] = {
+            commands.OptionType.USER: snowflakes.Snowflake,
+            commands.OptionType.CHANNEL: snowflakes.Snowflake,
```

Comment:
This doesn't need to be on the entity factory

Reply:
Moved it out :+1: